framework/generated/generated_vulkan_state_table.h

```diff
@@ -267,52 +267,421 @@ class VulkanStateTable : VulkanStateTableBase
+    std::unordered_map<format::HandleId, DescriptorPoolWrapper*> descriptorPool_map_;
+    std::unordered_map<format::HandleId, DescriptorSetWrapper*> descriptorSet_map_;
+    std::unordered_map<format::HandleId, DescriptorSetLayoutWrapper*> descriptorSetLayout_map_;
+    std::unordered_map<format::HandleId, DescriptorUpdateTemplateWrapper*> descriptorUpdateTemplate_map_;
+    std::unordered_map<format::HandleId, DeviceWrapper*> device_map_;
+    std::unordered_map<format::HandleId, DeviceMemoryWrapper*> deviceMemory_map_;
+    std::unordered_map<format::HandleId, DisplayKHRWrapper*> displayKHR_map_;
+    std::unordered_map<format::HandleId, DisplayModeKHRWrapper*> displayModeKHR_map_;
+    std::unordered_map<format::HandleId, EventWrapper*> event_map_;
+    std::unordered_map<format::HandleId, FenceWrapper*> fence_map_;
+    std::unordered_map<format::HandleId, FramebufferWrapper*> framebuffer_map_;
+    std::unordered_map<format::HandleId, ImageWrapper*> image_map_;
+    std::unordered_map<format::HandleId, ImageViewWrapper*> imageView_map_;
+    std::unordered_map<format::HandleId, IndirectCommandsLayoutNVWrapper*> indirectCommandsLayoutNV_map_;
+    std::unordered_map<format::HandleId, InstanceWrapper*> instance_map_;
+    std::unordered_map<format::HandleId, MicromapEXTWrapper*> micromapEXT_map_;
+    std::unordered_map<format::HandleId, OpticalFlowSessionNVWrapper*> opticalFlowSessionNV_map_;
+    std::unordered_map<format::HandleId, PerformanceConfigurationINTELWrapper*> performanceConfigurationINTEL_map_;
+    std::unordered_map<format::HandleId, PhysicalDeviceWrapper*> physicalDevice_map_;
+    std::unordered_map<format::HandleId, PipelineWrapper*> pipeline_map_;
+    std::unordered_map<format::HandleId, PipelineCacheWrapper*> pipelineCache_map_;
+    std::unordered_map<format::HandleId, PipelineLayoutWrapper*> pipelineLayout_map_;
+    std::unordered_map<format::HandleId, PrivateDataSlotWrapper*> privateDataSlot_map_;
+    std::unordered_map<format::HandleId, QueryPoolWrapper*> queryPool_map_;
+    std::unordered_map<format::HandleId, QueueWrapper*> queue_map_;
+    std::unordered_map<format::HandleId, RenderPassWrapper*> renderPass_map_;
+    std::unordered_map<format::HandleId, SamplerWrapper*> sampler_map_;
+    std::unordered_map<format::HandleId, SamplerYcbcrConversionWrapper*> samplerYcbcrConversion_map_;
+    std::unordered_map<format::HandleId, SemaphoreWrapper*> semaphore_map_;
+    std::unordered_map<format::HandleId, ShaderModuleWrapper*> shaderModule_map_;
+    std::unordered_map<format::HandleId, SurfaceKHRWrapper*> surfaceKHR_map_;
+    std::unordered_map<format::HandleId, SwapchainKHRWrapper*> swapchainKHR_map_;
+    std::unordered_map<format::HandleId, ValidationCacheEXTWrapper*> validationCacheEXT_map_;
+    std::unordered_map<format::HandleId, VideoSessionKHRWrapper*> videoSessionKHR_map_;
+    std::unordered_map<format::HandleId, VideoSessionParametersKHRWrapper*> videoSessionParametersKHR_map_;
```

Comment:
`WriteState()` might need adapting to use sort contents of `unordered_maps` if we use them since it iterates over maps and relied on the current order the `std::map` provides.

Reply:
Thank you. Fixed it in `Change map to unordered_map` commit.